core/app/models/spree/order.rb

```
@@ -907,5 +910,38 @@ def collect_payment_methods(store = nil)
     def credit_card_nil_payment?(attributes)
       payments.store_credits.present? && attributes[:amount].to_f.zero?
     end
+
+    def track_order_completed_event
+      analytics_event_handlers = Spree::Analytics.event_handlers.map do |handler|
+        handler.new(user: user, session: nil, request: nil, store: store)
+      end
+
+      analytics_event_handlers.each do |handler|
+        handler.handle_event('order_completed', {order: self})
```

RuboCop (1.73)

[convention] Space inside { missing.

(Layout/SpaceInsideHashLiteralBraces)